src/lib/Toast/Toast.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+  import { fly } from 'svelte/transition';
+  import { defaultToastProperties, type ToastProperties } from './properties';
+  import { createEventDispatcher, onDestroy, onMount } from 'svelte';
+
+  export let properties: ToastProperties = defaultToastProperties;
+  const dispatch = createEventDispatcher();
+  const colorMap: Record<string, string> = {
+    success: '#24aa5a',
+    error: '#f04438',
+    info: '#87ceeb',
+    warn: '#f3a42d'
+  };
+
+  let showToast = false;
+  let timeout: ReturnType<typeof setInterval>;
+  let backgroundColor =  properties.type !== undefined ? colorMap[properties.type] : '#87ceeb';
+
+  function hideToast() {
+    showToast = false;
+  }
+
+  function handleAnimationEnd() {
+    dispatch('handleAnimationEnd');
+  }
+
+  onMount(() => {
+    showToast = true;
+    timeout = setInterval(hideToast, properties.duration);
+  });
+
+  onDestroy(() => {
+    clearInterval(timeout);
+  });
+
+</script>
+
+{#if showToast}
+  <div
+    class="toast"
+    style="background-color: var(--toast-background-color, {backgroundColor})"
```

Comment:
@badri-singhal  Can these styles be inside toast class?

Reply:
we want to consider the below cases and for that we need a variable.

- If background color is passed then apply that background
- If it is not passed then apply on the basis of type
- If type is also not passed then apply a default info background color.